src/main/resources/liquibase/202405061230-remove-duplicated-hosts.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
```

Comment:
It would be worth noting this link somewhere, so that others can quickly see the dupe deletion queries are applying a common pattern: https://wiki.postgresql.org/wiki/Deleting_duplicates

Reply:
I didn't know about this link, good to be added +1